scripts/asm-installer/asm_vm

```diff
@@ -33,7 +33,7 @@ readonly ASM_META_VERSION
 readonly ASM_REVISION_PREFIX
 RELEASE=""
 
-SERVICE_PROXY_AGENT_BUCKET="${SERVICE_PROXY_AGENT_BUCKET:=gs://gce-service-proxy/service-proxy-agent/releases/service-proxy-agent-asm-${ASM_META_VERSION}-stable.tgz}"
```

Comment:
Is ASM_META_VERSION used anywhere else?

Reply:
Yes, https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/3d1fdddf81f1dbec5e94ab7b023ea112f59e67f9/scripts/asm-installer/asm_vm#L1249